pkg/trace/api/telemetry.go

```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/trace/metrics"
 )
 
+const FunctionARNKey = "function_arn"
```

Comment:
I think you can lowercase this const as it's only used in this package?

Reply:
Okay will do